sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/documentanalysis/implementation/models/AnalyzeResult.java

```diff
@@ -155,17 +157,6 @@ public List<DocumentParagraph> getParagraphs() {
         return this.paragraphs;
     }
 
-    /**
-     * Set the paragraphs property: Extracted paragraphs.
-     *
-     * @param paragraphs the paragraphs value to set.
-     * @return the AnalyzeResult object itself.
-     */
-    public AnalyzeResult setParagraphs(List<DocumentParagraph> paragraphs) {
```

Comment:
Removing public setter methods is a breaking change too.

Reply:
It should be fine as this in Impl model correct?